docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,6 @@
+##Contributing
+All boost repositories require DCO signing.
+By adding a signed off line to your commit, you sign off this [Developer Certificate of Origin](https://developercertificate.org).
+You can add the signed off line to your commit by adding the -s flag to your commit or have it
```

Comment:
Can we just link to this prepare-commit-msg somewhere as an idea instead of owning it?

Reply:
ok